enginetest/enginetests.go

```diff
@@ -1324,57 +1134,13 @@ func TestInsertErrorScriptsPrepared(t *testing.T, harness Harness) {
 	}
 }
 
-func TestScriptQueryPlan(t *testing.T, harness Harness) {
-	// TEST SCRIPTS
-	for _, script := range ScriptQueryPlanTest {
```

Comment:
In the process of making this use the regular helper funcs I realized `ScriptQueryPlanTest` was empy